src/App.tsx

```diff
@@ -3,12 +3,28 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  getFilteredMovies = () => {
+    const { query } = this.state;
+    const filteredMovies = moviesFromServer.filter(movie => (
+      (movie.title + movie.description).toLocaleLowerCase()
+        .includes(query.toLocaleLowerCase())
```

Comment:
query.toLocaleLowerCase() better to pass in a separate variable not to calculate it in every iteration